scan/lib/scan/options.rb

```diff
@@ -97,6 +97,12 @@ def self.available_options
                                      env_name: "SCAN_BUILDLOG_PATH",
                                      description: "The directory were to store the raw log",
                                      default_value: "#{FastlaneCore::Helper.buildlog_path}/scan"),
+        FastlaneCore::ConfigItem.new(key: :include_simulator_logs,
+                                     env_name: "SCAN_INCLUDE_SIMULATOR_LOGS",
+                                     description: "Should the logs of the app on the Simulator be written to the output_directory",
```

Comment:
These are the logs provided by the iOS app itself. For example, if you write `NSLog` statements in your app, these get written to the simulated device's system.log. You'll never see those log statements in your test output.

Super useful if you're trying to debug an issue that only occurs on a remote iOS CI slave and you absolutely need to know what is happening in the app.